lib/screens/home_screen.dart

```diff
@@ -64,6 +64,15 @@ class _HomeScreenState extends State<HomeScreen> {
     await TaskStorage.saveTasks(tasks);
   }
 
+  void _onOptionSelected(String option) {
+    setState(() {
+      if (option == "Delete all tasks") {
```

Comment:
Why not make this an enum? So that in fitute deleting multiple tasks but not all can be handled as well?

Reply:
True, will do it and update.

Reply:
Did that enum thing in last commit. I am not sure if this is what you meant @pranavkonidena . Please confirm.